testing/local-amp-chrome-extension/popup.html

```diff
@@ -123,20 +123,20 @@
     <script src="popup.js"></script>
   </head>
   <body>
-    <img id = "logo" src="logo.svg" />
+    <img id="logo" src="logo.svg" />
     <p class="desc">Switch between using a local distribution of AMP HTML or the online
     distribution hosted at https://cdn.ampproject.org/</p>
     <div id="switch-holder">
-      <label for="switch">Local AMP Serving</label>
-      <div id = "switch-wrapper">
-        <span id = "left">
+      <label id="switch-label" for="enabled">Local AMP Serving</label>
```

Comment:
As mentioned above, this doesn't need to be a `label` element (and it doesn't really make sense to have two labels for the same input), so let's switch this to a div with class `.section-header` or something similar to that

Reply:
IMO, it's fine to have multiple `label` elements that refer to the same input. It allows clicking the heading to toggle the checkbox, which would be consistent with clicking the heading for the base URL to focus. So I think this is fine?